inst/models/pk_pegasparaginase_modifiedwurthwein.json5

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "pk_pegasparaginase_wurthwein",
```

Comment:
We previously discussed using the `modified` tag since this model has been modified from the original. Did you want to include that in this version of the model?

Reply:
Great catch! Altered.

Reply:
we should rename this to be consistent with the file name
```suggestion
  "id": "pk_pegasparaginase_modifiedwurthwein",
```